test/test_jit.py

```diff
@@ -13148,9 +13150,7 @@ class TestJitGeneratedFunctional(JitTestCase):
     ('batch_norm', (S, S), (non_differentiable(torch.randn(S)), non_differentiable(torch.ones(S)), ),),
     ('instance_norm', (S, S, S), (non_differentiable(torch.zeros(S)), non_differentiable(torch.ones(S))),),
     ('layer_norm', (S, S, S, S), ([5],),),
-    ('layer_norm', (S, S, S, S), ([5], (S,)), 'with_only_weight'),
-    ('layer_norm', (S, S, S, S), ([5], None, (S,)), 'with_only_bias'),
-    ('layer_norm', (S, S, S, S), ([5], (S,), (S,)), 'with_weight_and_bias'),
```

Comment:
Why do we delete tests here?

Reply:
because for `nn.Layernorm`, we basically only calls with no weight and bias, or with both weight and bias, it does not make too much sense to add the tests to test `only_weight` and `only_bias`

Reply:
synced offline, it looks like we also expose nn.functional as public API so there will be cases people pass in only weight and bias, I will bring it back